### PR TITLE
Remove product settings video note

### LINKED ISSUE #5083: Inbox Tip: Learn more about Product Settings leads to outdated video on YouTube
**Is your feature request related to a problem? Please describe.**

I see that the `Learn more about Product Settings` Inbox tip was added not too long ago: https://github.com/woocommerce/woocommerce-admin/issues/4201:

tutorial

However, the video is from 2017 and outdated - I see that recent comments by users state the same: https://www.youtube.com/watch?v=FEmwJsE8xDY&t=. 

I realize that creating a new video is not a part of your responsibilities directly and likely updating the video is in the works by the appropriate team but I wanted to create an issue so that the video URL could be updated once there is a new one. 

**Describe the solution you'd like**

It will be great to update the URL of the YouTube video that the user is redirected to after clicking on the `Watch Tutorial` button on the `Learn more about Product Settings` Inbox tip card. 

**Describe alternatives you've considered**

N/A

**Should this be prioritized? Why?**

It will be nice to have. 

**Additional context**

N/A.

### PR DESCRIPTION
**Pull request by becdetat**
Fixes #5083

This removes the "Learn more about Product Settings" note, both from new stores (by deleting the note factory class) and from existing stores (by adding it as an obsolete note).

### Detailed test instructions:

- On a store with the "Learn more about Product Settings" note present, bump up the version in `FeaturePlugin.php` to trigger an install and delete the note
- Verify that the note has been removed
- Delete all of the existing notes and run the `wc_admin_daily` cron task
- Verify that the note has not been recreated

### Changelog Note:

Dev: Remove product settings video note

### CODE REVIEW

File: src/Install.php [hunk 1]
```diff
@@ -471,6 +471,7 @@ protected static function delete_obsolete_notes() {
 			'wc-admin-welcome-note',
 			'wc-admin-store-notice-setting-moved',
 			'wc-admin-store-notice-giving-feedback',
+			'wc-admin-learn-more-about-product-settings',
```
Comment (jeffstieler) [inline src/Install.php:474]: TIL!

**Review by jeffstieler — APPROVED** [review]
Looks good. :shipit:.

**Comment by timmyc** [comment]
Thanks for the review Jeff!

**Comment by timmyc** [comment]
quasi cherry picked in #5216

**Pull request merged**